data/avro-schema/src/main/resources/avro/net/corda/data/persistence/EntityResponse.avsc

```diff
@@ -16,6 +16,12 @@
       "name": "metadata",
       "doc": "Metadata returned from the execution of the persistence operation",
       "type": "net.corda.data.KeyValuePairList"
+    },
+    {
+      "name": "resumePoint",
+      "doc": "Used by queries that support stable paging to return opaque data that indicates where the next page should resume from",
+      "type": ["null", "bytes"],
```

Comment:
We don't want to be opinionated on the contents of the field, it could vary from query to query - the data is opaque from the perspective of the interface.